arangod/Transaction/Methods.cpp

```diff
@@ -1281,27 +1287,27 @@ Result transaction::Methods::documentFastPathLocal(std::string const& collection
   return res;
 }
 
-static Result resultFromClusterResult(std::shared_ptr<VPackBuilder> const& resultBody,
-                                      int defaultErrorCode) {
-  // read the error number from the response and use it if present
-  if (resultBody != nullptr) {
-    VPackSlice slice = resultBody->slice();
-    if (slice.isObject()) {
-      VPackSlice num = slice.get(StaticStrings::ErrorNum);
-      VPackSlice msg = slice.get(StaticStrings::ErrorMessage);
-      if (num.isNumber()) {
-        if (msg.isString()) {
-          // found an error number and an error message, so let's use it!
-          return Result(Result(num.getNumericValue<int>(), msg.copyString()));
-        }
-        // we found an error number, so let's use it!
-        return Result(num.getNumericValue<int>());
-      }
-    }
-  }
-
-  return Result(defaultErrorCode);
-}
+//static Result resultFromClusterResult(std::shared_ptr<VPackBuilder> const& resultBody,
```

Comment:
This seems to have been moved to Network/Utils. So let's remove the commented-out version here.